{
   "assignee" : null,
   "body" : "Running git HEAD, qa/rpc_tests/receivedby.py tells me:\r\n\r\n```\r\nAssertion failed: {u'account': u'', u'confirmations': 20, u'amount': Decimal('0.20000000')} : expected amount=0.30000000\r\n```\r\n\r\nI haven't tried to debug, it is most likely a missing synchronize-nodes-mempools in the test code.",
   "closed_at" : "2014-11-19T21:39:53Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/331997?v=3",
      "events_url" : "https://api.github.com/users/gavinandresen/events{/privacy}",
      "followers_url" : "https://api.github.com/users/gavinandresen/followers",
      "following_url" : "https://api.github.com/users/gavinandresen/following{/other_user}",
      "gists_url" : "https://api.github.com/users/gavinandresen/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/gavinandresen",
      "id" : 331997,
      "login" : "gavinandresen",
      "organizations_url" : "https://api.github.com/users/gavinandresen/orgs",
      "received_events_url" : "https://api.github.com/users/gavinandresen/received_events",
      "repos_url" : "https://api.github.com/users/gavinandresen/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/gavinandresen/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/gavinandresen/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/gavinandresen"
   },
   "comments" : 0,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/5305/comments",
   "created_at" : "2014-11-18T19:29:19Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/5305/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/5305",
   "id" : 49282330,
   "labels" : [],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/5305/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "number" : 5305,
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "title" : "receivedby.py regression test failing",
   "updated_at" : "2014-11-19T21:39:53Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/5305",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/331997?v=3",
      "events_url" : "https://api.github.com/users/gavinandresen/events{/privacy}",
      "followers_url" : "https://api.github.com/users/gavinandresen/followers",
      "following_url" : "https://api.github.com/users/gavinandresen/following{/other_user}",
      "gists_url" : "https://api.github.com/users/gavinandresen/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/gavinandresen",
      "id" : 331997,
      "login" : "gavinandresen",
      "organizations_url" : "https://api.github.com/users/gavinandresen/orgs",
      "received_events_url" : "https://api.github.com/users/gavinandresen/received_events",
      "repos_url" : "https://api.github.com/users/gavinandresen/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/gavinandresen/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/gavinandresen/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/gavinandresen"
   }
}
